<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VqbInstancesConstruct</class>
 <widget class="QWidget" name="VqbInstancesConstruct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxOutputs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Outputs</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBoxOTriples">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Triple</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelSOutputs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="KComboBox" name="cbSubjectOutputs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="statusTip">
            <string>Subject Field</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPOutputs">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Predicate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="KComboBox" name="cbPredicateOutputs">
           <property name="statusTip">
            <string>Predicate Field</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelOOutputs">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Object</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="KComboBox" name="cbObjectOutputs">
           <property name="toolTip">
            <string>Enter a string, integer, date or URI.

Examples:
	foaf:Person
	5
	Knud Moeller
            	&lt;http://purl.org/dc/terms/title&gt;
	</string>
           </property>
           <property name="statusTip">
            <string>Object field</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="KComboBox" name="cbTypeOutputs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="statusTip">
            <string>Object (Literal) Type</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="KEditListBox" name="listBoxOutputs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Output Triples</string>
        </property>
        <property name="title">
         <string>Graph Pattern</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxConditions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Conditions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBoxCTriples">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Enter a string, integer, date or URI.

Examples:
	foaf:Person
	5
	Knud Moeller
            	&lt;http://purl.org/dc/terms/title&gt;
	</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Triple</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="labelSConditions">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="KComboBox" name="cbSubjectConditions">
           <property name="statusTip">
            <string>Subject Field</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPConditions">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Predicate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="KComboBox" name="cbPredicateConditions">
           <property name="statusTip">
            <string>Predicate Field</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBoxFilterConditions">
           <property name="statusTip">
            <string>Filter on Object Value</string>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="KComboBox" name="cbRelationConditions">
           <property name="statusTip">
            <string>Filter Relation</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelOConditions">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string>Object</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="KComboBox" name="cbObjectConditions">
           <property name="statusTip">
            <string>Object Field</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="KComboBox" name="cbTypeConditions">
           <property name="statusTip">
            <string>Object (Literal) Type</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="KEditListBox" name="listBoxConditions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>3</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Conditional Triples</string>
        </property>
        <property name="title">
         <string>Graph Pattern</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbSubjectOutputs</tabstop>
  <tabstop>cbPredicateOutputs</tabstop>
  <tabstop>cbObjectOutputs</tabstop>
  <tabstop>cbTypeOutputs</tabstop>
  <tabstop>cbSubjectConditions</tabstop>
  <tabstop>cbPredicateConditions</tabstop>
  <tabstop>checkBoxFilterConditions</tabstop>
  <tabstop>cbRelationConditions</tabstop>
  <tabstop>cbObjectConditions</tabstop>
  <tabstop>cbTypeConditions</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>updateTriple()</slot>
 </slots>
</ui>
